pkg/api/interface.go

```diff
@@ -44,17 +44,41 @@ package api
 
 const ServerPatchAnnotationName = "dual-pod.llm-d.ai/server-patch"
 
+// InferenceServerContainerName is the name of the container which is described by the server patch.
+// This container is expected to run the inference server using vLLM.
+const InferenceServerContainerName = "inference-server"
+
+// ServerRunningPodNameSuffix is the suffix that is appended to the name of the
+// server-requesting Pod to form the name of the server-running Pod.
+const ServerRunningPodNameSuffix = "-server"
+
 // AdminPortAnnotationName is the name of an annotation whose value
-// is the name of the port on the "inference-server" container to be
+// is the name of the port on the server-requesting pod to be
```

Comment:
In a PodSpec, ports are declared on a container, right?

Reply:
Yes, changed.